src/main/java/constants/UiConstant.java

```diff
@@ -20,6 +20,9 @@ public class UiConstant {
     public static final String PARTITION_LINE = "____________________________________________________________";
     public static final String EMPTY_STRING = "";
     public static final String FULL_STOP = ".";
+    public static final String NEW_LINE = "%n";
```

Comment:
Either use `System.lineSeparator())` or we implement this as NEW_LINE and have to apply and use it across the board. Let me know your thoughts

Reply:
Sure